src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-sessions-table/tutorial-group-session-row/tutorial-group-session-row.component.ts

```diff
@@ -24,6 +29,15 @@ export class TutorialGroupSessionRowComponent implements OnChanges {
 
     @Input() timeZone?: string = undefined;
 
+    @Input() isReadOnly = false;
+
+    @Output() attendanceChanged = new EventEmitter<TutorialGroupSession>();
+
+    persistedAttendanceCount?: number | undefined = undefined;
```

Comment:
Isn't the | undefined already given by the `?`?

Reply:
Done in 74c048acc4650855f58452768fe65a2fbbd46130